### PR TITLE
python310Packages.pyjson5: remove duplicate package, python310Packages.json5: 0.9.6 -> 0.9.9

### PR DESCRIPTION
**Pull request by anthonyroussel**
###### Description of changes

* Remove duplicate package `python310Packages.pyjson5`
  * Replaced by the most recent one: `python310Packages.json5`

* Upgrade `python310Packages.json5` from 0.9.6 -> 0.9.9

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by anthonyroussel** [comment]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab-lsp</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
  </ul>
</details>
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>manim</li>
    <li>python310Packages.graphtage</li>
    <li>python310Packages.json5</li>
    <li>python310Packages.jupyterlab</li>
    <li>python310Packages.jupyterlab-git</li>
    <li>python310Packages.jupyterlab-lsp</li>
    <li>python310Packages.jupyterlab_server</li>
    <li>python310Packages.jupytext</li>
    <li>python310Packages.webdav4</li>
    <li>python310Packages.wsgidav</li>
    <li>python39Packages.graphtage</li>
    <li>python39Packages.json5</li>
    <li>python39Packages.webdav4</li>
    <li>python39Packages.wsgidav</li>
  </ul>
</details>

Looks like jupyterlab for Python 3.9 is broken.

```console
> ERROR: Could not find a version that satisfies the requirement importlib-metadata>=3.6; python_version < "3.10" (from jupyterlab-server) (from versions: none)
> ERROR: No matching distribution found for importlib-metadata>=3.6; python_version < "3.10"
> 
```

The error is unrelated to the pyjson5/json5 changes.

**Comment by risicle** [comment]
The `throw` probably belongs in `python-aliases.nix`.

**Comment by anthonyroussel** [comment]
> The `throw` probably belongs in `python-aliases.nix`.

@risicle Thanks for the review.
I moved the `throw` to `python-aliases.nix`

File: pkgs/top-level/python-aliases.nix [hunk 1]
```diff
@@ -126,6 +126,7 @@ mapAliases ({
   pycryptodome-test-vectors = throw "pycryptodome-test-vectors has been removed because it is an internal package to pycryptodome"; # added 2022-05-28
   pyialarmxr = pyialarmxr-homeassistant; # added 2022-06-07
   pyialarmxr-homeassistant = throw "The package was removed together with the component support in home-assistant 2022.7.0"; # added 2022-07-07
+  pyjson5 = throw "pyjson5 has been renamed to/replaced by json5"; # added 2022-08-28
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-aliases.nix:129]:
```suggestion
  pyjson5 = json5; # added 2022-08-28
```

Comment (anthonyroussel) [reply pkgs/top-level/python-aliases.nix]: Thanks for the review. Fixed

**Comment by risicle** [comment]
On macos 10.15 I'm getting

```
FAILED tests/lib_test.py::TestDumps::test_object_roundtrip - hypothesis.error...
```

for `json5` on python 3.9 & 3.10. This might be timeout-related.

**Comment by risicle** [comment]
@ofborg build python39Packages.json5 python310Packages.json5

**Comment by risicle** [comment]
On a less loaded system it does indeed build for me on macos 10.15 now.

**Review by risicle — APPROVED** [review]
`nixpkgs-review` reveals no new failures, macos 10.15 & nixos x86_64

**Pull request merged**